src/index.js

```diff
@@ -0,0 +1 @@
+require('../vendor/aframe.min.js');
```

Comment:
Instead of keeping this in your repository, you could install aframe via npm or yarn, and pull it in as a dependency:

```bash
npm install --save aframe
```

```javascript
// in index.js
require('aframe');
```